recipes-core/initrdscripts/files/init-readonly-rootfs-overlay-boot.sh

```diff
@@ -61,6 +61,10 @@ read_args() {
 				ROOT_RWRESET=$optarg ;;
 			rootrwoptions=*)
 				ROOT_RWMOUNTOPTIONS_DEVICE="$optarg" ;;
+			overlaymodule=*)
```

Comment:
Now I have time for a more proper review.

Do you think 'overlayfstype' is a better name? To be consistent with rootfstype and rootrwfstype.

Could you also please add the command line argument to the Details section in README.md?

Reply:
yeah I was thinking since it only loaded the module it wouldn't make sense to imply something broader, but you are right it would be more consistent if it was type.  Let me change that quick